src/app/shell/admin-tools/platform/platform-info/info-edit/info-edit.component.ts

```diff
@@ -40,6 +42,7 @@ export class InfoEditComponent extends CreateFormComponent implements OnInit, On
   platformInfo: CompanyInformation;
 
   platformInfoType: AdminTabsTitle;
+  isMainPage: boolean = false;
```

Comment:
you don't need to initialize this value as false, because isMainPage by default is false